src/pages/project/project_wizard/CreateNfrProject.tsx

```diff
@@ -34,7 +37,7 @@ export const CreateNfrProject = ({
 
     setSuggestedProjectManager(selectedProject.lead);
 
-    setNewProject({
+    const newNFRProject: SaveCristinProject = {
```

Comment:
Bruker stort sett bare en stor bokstav for lignende ting ellers
```suggestion
    const newNfrProject: SaveCristinProject = {
```